src/InteractiveWindow/Version.props

```diff
@@ -0,0 +1,5 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <RoslynSemanticVersion>1.0.0</RoslynSemanticVersion>
```

Comment:
We need to version Interactive Window separately from Roslyn. Change in IW version requires change in PTVS.